examples/Hello_World/hello_world.py

```diff
@@ -11,7 +11,7 @@
 
 # Create a log file.
 logger = logging.getLogger()
-logger.addHandler(logging.StreamHandler(file("hello_world.log", "w")))
+logger.addHandler(logging.StreamHandler(open("hello_world.log", "w")))
```

Comment:
Similar to @kitchoi 's comment on #319, we probably want `FileHandler` here.

If the `open` stays, please add the encoding parameter. As a general rule, all uses of `open` should have that parameter explicitly specified, else potentially odd system-dependent things happen on Windows.